lib/features/upload/presentation/controller/upload_controller.dart

```diff
@@ -170,9 +170,6 @@ class UploadController extends BaseController {
             cid: uuid.v1()
           );
 
-          final uploadFileState = _uploadingStateInlineFiles.getUploadFileStateById(success.uploadId);
-          log('UploadController::_handleProgressUploadInlineImageStateStream:uploadId: ${uploadFileState?.uploadTaskId} | fromFileShared: ${uploadFileState?.file?.isShared}');
-
```

Comment:
why we need to remove it?

Reply:
`final uploadFileState` is just for logging purposes, we don't need it. Avoid wasting memory